.github/workflows/regression.yml

```diff
@@ -87,7 +87,8 @@ jobs:
       - name: Run Tests in Container
         uses: docker://su2code/test-su2:20200303
         with:
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          # -t <Tutorials-branch> -c <Testcases-branch>
+          args: -b ${{github.ref}} -t feature_heatcoeff_BC -c develop -s ${{matrix.testscript}}
```

Comment:
a little service to future me (and anyone who wonders) ;)

Reply:
:+1: Kudos on resisting the "stale bot uprising" btw.

Reply:
Reminder to revert

Reply:
Hm, I of course have to merge su2code/Tutorials/pull/16 first -> revert this line -> merge this PR in order to make it work.
Unfortunately this will break everyone's regression tests that do not merge develop immediately :( If I am not mistaken

Reply:
Yep just make the change to KT_CONSTANT and then update tutorials once you are ready to merge.

Reply:
👍